src/pyairios/models/brdg_02r13.py

```diff
@@ -203,14 +234,118 @@ def __init__(self, slave_id: int, client: AsyncAiriosModbusClient) -> None:
             U16Register(Reg.ADDRESS_NODE_32, RegisterAccess.READ),
         ]
         self._add_registers(brdg_registers)
+        # models are lazy loaded or initiated by caller
 
     def __str__(self) -> str:
         return f"BRDG-02R13@{self.slave_id}"
+        # node method doesn't work for Bridge module in CLI (contains the path too)
+
+    async def load_models(self) -> int:
```

Comment:
This is a "library" functionality not related bridge device. There are other bridge models like BRDG-02EM23 which is Ethernet instead of RS485. I would move supported models loading to `class Airios`